libs/output-mapping/tests/InitSynapseStorageClientTrait.php

```diff
@@ -4,26 +4,11 @@
 
 namespace Keboola\OutputMapping\Tests;
 
-use Keboola\StorageApi\Exception;
 use Keboola\StorageApiBranch\ClientWrapper;
 use Keboola\StorageApiBranch\Factory\ClientOptions;
 
 trait InitSynapseStorageClientTrait
 {
-    protected function checkSynapseTests(): bool
-    {
-        if (!getenv('RUN_SYNAPSE_TESTS')) {
-            return false;
-        }
-        if (getenv('SYNAPSE_STORAGE_API_TOKEN') === false) {
-            throw new Exception('SYNAPSE_STORAGE_API_TOKEN must be set for synapse tests');
-        }
-        if (getenv('SYNAPSE_STORAGE_API_URL') === false) {
-            throw new Exception('SYNAPSE_STORAGE_API_URL must be set for synapse tests');
-        }
-        return true;
```

Comment:
staci ze se to kontroluje uz v bootstrapu
https://github.com/keboola/platform-libraries/blob/1345cbe90fa9aaa6c012bd9bd051f5b0acc7b1de/libs/output-mapping/tests/bootstrap.php#L21-L27